src/connection_string.ts

```diff
@@ -36,7 +37,7 @@ function matchesParentDomain(srvAddress: string, parentDomain: string): boolean
  * @param options - Optional user provided connection string options
  */
 function parseSrvConnectionString(uri: string, options: any, callback: Callback) {
-  const result: any = url.parse(uri, true);
+  const result: Document = url.parse(uri, true);
```

Comment:
```suggestion
  const result: ArbitraryOptions = url.parse(uri, true);
```

Reply:
also bring on the :bike: :house: but should it really be called `AnyOptions`?

Reply:
I'm fine with that, you could say the name is... _arbitrary_ 🥁